recipe/meta.yaml

```diff
@@ -36,13 +30,27 @@ requirements:
   run:
     - python
 
+{% set skip_tests = [
+    [aarch64 or ppc64le, 'test_loss_of_precision',              'Seems to fail on aarch64 and ppc64le'],
+    [ppc64le,            'test_einsum_sums_cfloat64',           'expected failure ppc64le'],
+    [aarch64,            'test_may_share_memory_easy_fuzz',     'Times out on qemu+aarch'],
+    [aarch64,            'test_xerbla_override',                'Seems to timeout on aarch64'],
+    [aarch64,            'test_may_share_memory_harder_fuzz',   'Seems to timeout on aarch64'],
+    [aarch64,            'test_large_zip',                      'Seems to timeout on aarch64'],
+    [aarch64,            'test_sdot_bug_8577',                  'hangs on aarch64'],
+  ]
+%}
+
 test:
   requires:
     - pytest
     - pytest-timeout
   commands:
     - f2py -h
-    - pytest --timeout=300 -v --pyargs numpy
+    - pytest --timeout=300 -v --pyargs numpy -k "not (conda_forge
+{%- for skip_test in skip_tests -%}
+{%- if skip_test[0] %} or {{ skip_test[1] }}{%- endif -%}
+{%- endfor -%})"
```

Comment:
@conda-forge/numpy, what do you think about this slightly complicated jinja?

Reply:
I was just going to complain about this. I can't understand anything that is going on. 

What about drone? Do tests pass on that without patching? If so, bugs should be submitted to qemu

Reply:
This skips the tests in the `skip_tests` list based on the first condition.

Reply:
> @conda-forge/numpy, what do you think about this slightly complicated jinja? Should we go back to patching?

I kind of prefer patching.